src/pages/inventory/[[...section]].tsx

```diff
@@ -547,7 +547,7 @@ const Inventory = () => {
   const [data, totals, updates, emptyMessage] = useMemo(() => {
     const empty: Record<string, NonNullable<Nft["listing"]>> = {};
     const {
-      hidden = [],
+      staked = [],
```

Comment:
In order for this change to work, you'd have to change the GraphQL query. I think for now we can keep this as hidden.

```suggestion
      hidden = [],
```